core/lib/multivm/src/versions/era_vm/vm.rs

```diff
@@ -59,6 +62,7 @@ pub struct Vm<S: ReadStorage> {
     pub(crate) inner: EraVM,
     pub suspended_at: u16,
     pub gas_for_account_validation: u32,
+    pub world: World<S>,
```

Comment:
It might be a good idea to remove the storage and program_cache fields from the Vm since we already have them in world. I know this refactor might be a little bit tricky since it would require changing all the current tests and many other parts of the code. Anyway, I'm leaving this comment in case you'd like to add it here; otherwise, we could open a new issue!